src/WebJobs.Script/Description/Workers/WorkerFunctionDescriptorProvider.cs

```diff
@@ -90,5 +102,37 @@ protected override IFunctionInvoker CreateFunctionInvoker(string scriptFilePath,
 
             return parameters;
         }
+
+        private bool BindingAttributeContainsExpression(IEnumerable<FunctionBinding> bindings)
+        {
+            if (bindings.Count() <= 0)
```

Comment:
can remove this check once you make the regex a field

Reply:
I'm not sure I follow; what does the regex field have to do with checking the binding.Count?

Reply:
I was just pointing out that you don't need to do a .Count() here at all - the below foreach will only execute if there are actually any bindings. As it stands, you're doing an unnecessary double enumeration. I had mentioned the regex because previously you were initializing it here before the foreach, and I thought you were trying to avoid that instantiation.

Reply:
Ahhh gotcha, that makes sense - thanks!